docker/docs/Dockerfile

```diff
@@ -9,7 +9,7 @@ MAINTAINER Rob Kooper <kooper@illinois.edu>
 
 RUN apt-get update \
     && apt-get install -y --no-install-recommends pandoc \
-    && install2.r -e -s -n -1 bookdown \
+    && install2.r -e -s -r cloud.r-project.org -n -1 bookdown \
```

Comment:
I'm not entirely happy with this hack -- It feels a little like a step backwards from always using the repositories set by Rocker. But since this image just serves the compiled docs without needing to interact with the rest of the system, it feels simplest to jump straight to the latest CRAN version of bookdown here.